src/ir/module.h

```diff
@@ -85,6 +96,11 @@ class Block {
 
   Block() : parent_module_(nullptr) {}
 
+  Block(OperationList operations, NamedValueMap results)
+      : parent_module_(nullptr),
+        operations_{std::move(operations)},
+        results_{results} {}
```

Comment:
`results_{std::move(results)}`?

Reply:
I thought this was one of those cases where the name can't be used after this assignment so there's no need for a move, but I can put it in :)